src/Components/MyEventPage.js

```diff
@@ -1,13 +1,149 @@
 import React, { Component } from 'react';
+import { MDBContainer, MDBRow, MDBCol, MDBCard, MDBCardBody, MDBTabPane, MDBTabContent, MDBNav, MDBNavItem, MDBNavLink } from "mdbreact";
+import EventCard from './EventCard.js'
+import Images from "../images.js"
+import axios from "axios";
 import './styles/MyEventPage.css'
 
+const eventTemplates = [
```

Comment:
Can you add a `//TODO: remove after backend integration` here. Or tbh, if you have axios and can integrate it, lets just remove the fake data moving onwards.